app/views/booking_requests/step_two.html.erb

```diff
@@ -130,7 +130,7 @@
           <% if @booking_request.errors.include?(:accessibility_requirements) %>
             <span class="error-message"><%= @booking_request.errors[:accessibility_requirements].to_sentence %></span>
           <% end %>
-          <%= f.check_box :accessibility_requirements, class: 't-accessibility-requirements' %> I have accessibility needs
+          <%= f.check_box :accessibility_requirements, class: 't-accessibility-requirements' %> I have accessibility needs <span style="color:#6f777b">eg wheelchair access</span>
```

Comment:
That style thing shouldn't be declared inline but I'll defer to @benbarnett or @Guntrisoft.